src/tribol/mesh/MfemData.hpp

```diff
@@ -45,8 +45,9 @@ class SubmeshLORTransfer {
    * @param submesh_fes Higher order finite element space on the parent-linked
    * boundary submesh
    * @param lor_mesh LOR mesh
+   * @param use_ea Whether to use device-friendly element assembly for the transfer (default: false)
```

Comment:
Perhaps elaborate on what device-friendly means here. If we are running common plane on device, will we typically want this to be `true` for HO + LOR?